lib/app/views/teams/member_submissions.erb

```diff
@@ -25,7 +25,7 @@
   </p>
   <% if submissions.any? %>
     <ul>
-      <% submissions.each do |submission| %>
+      <% submissions.limit(5).each do |submission| %>
```

Comment:
Hm, good point. I assumed submissions were returned in a specific order, but that's a terrible thing to assume. I'll fix that.